src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -40,12 +40,12 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     /*******************************************************************************************************
     * @description Map of Contact field names to Contact field labels.
     ********************************************************************************************************/
-    public Map<String, String> contactLabelNames;
+    public Map<String, String> contactFieldAPINamesByLabels;
```

Comment:
Very minor suggestion, might be more clear if we name it contactFieldAPINameByLabel instead since it is a 1:1 mapping.